tests/dummy/app/router.js

```diff
@@ -3,7 +3,7 @@ const {Router} = Ember
 
 import config from './config/environment'
 
-var MyRouter = Router.extend({
+const MyRouter = Router.extend({
```

Comment:
Should remain `var` to remain consistent with all the other `ember-cli 2.12.3` installations.